scripts/setup/install_aws_dependencies.sh

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env bash
+
+#
+# MIT License
+#
+# Copyright (c) 2024 HySCALE
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+LOADER_NODE=$1
+GO_VERSION=1.21.9
+
+server_exec() {
+  ssh -oStrictHostKeyChecking=no -p 22 $LOADER_NODE $1;
+}
+
+# Check the architecture
+arch=`server_exec 'uname -m'`
+case $arch in
+  'x86_64')
+    arch='amd64'
+    ;;
+  'aarch64')
+    arch='arm64'
+    ;;
+  *)
+    echo "Unsupported architecture $arch"
+    exit 1
+    ;;
+esac
+
+# Obtain configuration variables
+DIR=$(cd "$(dirname ${BASH_SOURCE[0]})" > /dev/null 2>&1 && pwd)
+source "$DIR/setup.cfg"
+LOADER_BRANCH=$(echo -e $LOADER_BRANCH | sed -e 's/[[:space:]]*$//')  # Remove trailing whitespace (i.e. convert main? to main)
```

Comment:
Do you have a specific reason to do that? I've not encountered this kind of problem while using the create_mulitnode.sh. The version is specified (by default) in quotes, it's pretty hard to accidentally put there that would have trailing symbols.